webproject/src/main/java/data/Room.java

```diff
@@ -0,0 +1,52 @@
+package data;
+
+import java.io.File;
+
+/**
+ * This class defines room objects
+ */
+public class Room {
+  private String id;
+  private String title;
+  private String link;
+  private String description;
+  private int deliveryLocation; //modify to use different representation of location if needed
+  private int phoneNumber; //modify to use different representation of contact details if needed
+  private String category;
+  private String imagePath;
+  private RoomType roomType;
+  private RoomCondition roomCondition;
+
+  public Room (String id, String title, String link, String description, int deliveryLocation, int phoneNumber, String category, String imagePath, RoomType roomType, RoomCondition roomCondition) {
```

Comment:
lets change this to builder class
https://dzone.com/articles/design-patterns-the-builder-pattern

Reply:
Changed!